attrs_strict/__init__.pyi

```diff
@@ -0,0 +1,2 @@
+from ._error import AttributeTypeError as AttributeTypeError, BadTypeError as BadTypeError, TupleError as TupleError, TypeValidationError as TypeValidationError, UnionError as UnionError
```

Comment:
we need some tool that validates that content from the python file matches with the stubs; for example https://pypi.org/project/retype/ is an example, please use it in our tox check...  see https://github.com/ambv/retype/blob/master/tox.ini#L58

Reply:
wasn't aware of this. will add!